### PR TITLE
[ZT] Windows 11 and Server 2022 DoH procedure

### PR DESCRIPTION
**Pull request by maxvp**

### CODE REVIEW

File: content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md [hunk 1]
```diff
@@ -30,6 +30,78 @@ Browsers can be configured to use any DNS over HTTPS (DoH) endpoint. If you choo
 
 Your DNS queries will now be sent to Gateway for filtering. To filter these requests, build a DNS policy using the [**DNS Location**](/cloudflare-one/policies/filtering/dns-policies/#dns-location) selector.
 
+### Configure opearting system for DoH
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md]:
```suggestion
### Configure operating system for DoH
```

File: content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md [hunk 2]
```diff
@@ -30,6 +30,78 @@ Browsers can be configured to use any DNS over HTTPS (DoH) endpoint. If you choo
...
+
+<details>
+<summary>Windows 11</summary>
+<div>
+
+1. Obtain the A and AAAA record values associated with your location's DoH endpoint.
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md]:
```suggestion
1. Obtain the `A` and `AAAA` record values associated with your location's DoH endpoint.
```

Comment (ranbel) [reply content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md]: DNS record types are monospaced according to our style guide: https://wiki.cfdata.org/display/Prod/Code+Conventions+and+Formatting

File: content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md [hunk 3]
```diff
@@ -30,6 +30,78 @@ Browsers can be configured to use any DNS over HTTPS (DoH) endpoint. If you choo
...
+
+   1. Run the following command to obtain your A record values:
+
+   ```sh
+   $ nslookup -type=A <your-subdomain>.cloudlfare-gateway.com
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md]:
```suggestion
   $ nslookup -type=A <your-subdomain>.cloudflare-gateway.com
```

File: content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md [hunk 4]
```diff
@@ -30,6 +30,78 @@ Browsers can be configured to use any DNS over HTTPS (DoH) endpoint. If you choo
...
+
+   ```sh
+   $ nslookup -type=AAAA <your-subdomain>.cloudlfare-gateway.com
+   ```
+
+   3. Copy the resulting IP addresses.
+
+2. Add the addresses to your list of known DoH servers.
+
+   1. In an elevated terminal, run the following command for each address:
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md]: what does elevated mean in this context?

File: content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md [hunk 5]
```diff
@@ -30,6 +30,78 @@ Browsers can be configured to use any DNS over HTTPS (DoH) endpoint. If you choo
...
+<details>
+<summary>Windows Server 2022</summary>
+<div>
+
+Obtain the A and AAAA record values associated with your location's DoH endpoint.
+
+1. Run the following command to obtain your A record values:
+
+```sh
+$ nslookup -type=A <your-subdomain>.cloudlfare-gateway.com
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md]:
```suggestion
$ nslookup -type=A <your-subdomain>.cloudflare-gateway.com
```

File: content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md [hunk 6]
```diff
@@ -30,6 +30,78 @@ Browsers can be configured to use any DNS over HTTPS (DoH) endpoint. If you choo
...
+```
+
+2. Obtain your AAAA record values.
+
+```sh
+$ nslookup -type=AAAA <your-subdomain>.cloudlfare-gateway.com
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md]:
```suggestion
$ nslookup -type=AAAA <your-subdomain>.cloudflare-gateway.com
```

File: content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md [hunk 7]
```diff
@@ -30,6 +30,78 @@ Browsers can be configured to use any DNS over HTTPS (DoH) endpoint. If you choo
...
+```
+
+3. Copy the resulting IP addresses.
+
+Next, use these values in [Microsoft's DoH guide](https://learn.microsoft.com/en-us/windows-server/networking/dns/doh-client-support) for Windows Server 2022 and newer.
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md]: I think we could be more explicit here.
Step 4: Add address to list of known DoH servers: https://learn.microsoft.com/en-us/windows-server/networking/dns/doh-client-support#add-a-new-doh-server-to-the-list-of-known-servers
Step 5: Configure the DNS client to support DoH: https://learn.microsoft.com/en-us/windows-server/networking/dns/doh-client-support#configure-the-dns-client-to-support-doh

File: content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md (hunk 4 above)
Comment (maxvp) [reply content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md]: Elevated = **Run as Administrator**. I think the consensus is to refer to it as "elevated", but we can explicitly call out running as an admin if need be.

File: content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md [hunk 8]
```diff
@@ -30,6 +30,78 @@ Browsers can be configured to use any DNS over HTTPS (DoH) endpoint. If you choo
...
+9. In **Preferred DNS** and **Alternate DNS**, enter the IPv6 addresses from your AAAA record command. Set **DNS over HTTPS** to _On (automatic template)_.
+
+</div>
+</details>
+
+<details>
+<summary>Windows Server 2022</summary>
+<div>
+
+Obtain the A and AAAA record values associated with your location's DoH endpoint.
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md]: Add backticks to DNS record types throughout the doc.

File: content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md (hunk 4 above)
Comment (ranbel) [reply content/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https.md]: I think our docs assume that all commands are run as an admin.

**Review by ranbel — APPROVED** [review]

**Pull request merged**